Functions/xASL_stat_PrintStats.m

```diff
@@ -109,15 +109,26 @@
 %% 3) Define number of sessions, force to 1 in case of TT or volume metrics
 SingleSessions = {'volume' 'TT' 'PV_pGM' 'PV_pWM' 'pGM' 'pWM' 'mrc1T1' 'mrc2T1'};
 
-HasSingleSessionOnly = sum(cellfun(@(y) ~isempty(findstr(y,x.S.output_ID)), SingleSessions));
+HasSingleSessionOnly = sum(cellfun(@(y) ~isempty(regexp(y,['^' x.S.output_ID])), SingleSessions));
 
 if HasSingleSessionOnly
     nSessions = 1;
 else
-    FileList = xASL_adm_GetFileList(x.D.PopDir,'.*ASL_\d\.nii$');
-    [~, Ffile] = cellfun(@(x) xASL_fileparts(x),FileList, 'UniformOutput',false);
-    SessionsN = unique(cellfun(@(x) x(end-4:end),Ffile, 'UniformOutput',false));
-    nSessions = length(SessionsN);
+    SessionList = xASL_adm_GetFileList(x.D.PopDir,['^' x.S.InputDataStr '.*_ASL_\d*\.nii'], 'FPList', [0 Inf]);
```

Comment:
This is the same code in `xASL_stat_GetROIstatistics` right? That would break the modularity of the code. The idea is that `xASL_stat_PrintStats` only prints stuff that it gets from another function/wrapper, whereas e.g. `xASL_stat_GetROIstatistics` or `xASL_wrp_GetROIstatistics` is responsible for loading the data. Should be clear in the header, otherwise perhaps I need to make this clear ;)

Reply:
Okay so we want to create a new cell in the x structure that feeds into xASL_stat_PrintStats?

Reply:
Is all this here only to remove the NIfTIs with suffixes `_masked`? There's an easier solution for that.
Isn't `_masked` used somewhere? `qCBF_masked` have vascular peaks replaced by NaNs, so it will limit the impact of vascular outliers/artifacts on regional CBF estimates. For sCoV you don't want this but for CBF this can be helpful.

Reply:
So we should look for `_masked` instead? If for some reason there is no `_masked` then the code will show NaN's, there should also be a warning then?